app/oauth.go

```diff
@@ -517,6 +517,13 @@ func (a *App) RegenerateOAuthAppSecret(app *model.OAuthApp) (*model.OAuthApp, *m
 func (a *App) RevokeAccessToken(token string) *model.AppError {
 	session, _ := a.GetSession(token)
 
+	defer func() {
```

Comment:
Can we refactor this into a separate function?

Reply:
Refactored into a separate function, I will keep this comment open, let me know if you need any changes on this.